### DISCUSSION TITLE
How can I set preview image for a static page in docusaurus?

### DISCUSSION THREAD

**Discussion by pranay01** [facebook/docusaurus#5866] [Q&A] [answered]
I want to set preview image for a static page which is different from the generic preview image set in docusaurus-config.js - Is there a way to do this? The static page is not a .md page but js page. -  so can't use https://github.com/facebook/docusaurus/pull/5375 i guess

if I can set page specifc
meta property="og:image" - then this should solve the issue

Current the specific page takes site wide preview image.

**Comment by Josh-Cena** [comment, marked as answer]
You can use the `<Head>` component: https://docusaurus.io/docs/next/docusaurus-core#head

E.g.:

```js
import React from 'react';
import Head from '@docusaurus/Head';

export default function page() {
  return (
    <main>
      <Head>
        <meta property="og:image" content="image.png" />
      </Head>
      {/* ... */}
    </main>
  );
}

**Reply by slorber** [reply]
It also works in markdown docs/blog posts: https://docusaurus.io/docs/next/markdown-features/head-metadatas

**Reply by vnugent** [reply]
@slorber this feature is documented in v2, but you need to specify it for each each page.  Is there a global og:image setting?

**Reply by Josh-Cena** [reply]
@vnugent Yes: https://docusaurus.io/docs/api/themes/configuration#meta-image

**Reply by vanshika-srivastava** [reply]
I did this but still the image doesn't shows up.

**Comment by jcstein** [comment]
how can this be done on a markdown page? with `image: file/path` in frontmatter?

**Reply by slorber** [reply]
We don't have an `image` front matter for the page plugin:
https://docusaurus.io/docs/next/api/plugins/@docusaurus/plugin-content-pages#markdown-front-matter

But you can use the `<head>` component as explained above

**Reply by jcstein** [reply]
Awesome, thank you @slorber. Cc @alex-beckett

**Reply by jcstein** [reply]
what about on a markdown page @slorber?

**Reply by slorber** [reply]
@jcstein the suggestion solution above is for a markdown page:

https://docusaurus.io/docs/markdown-features/head-metadata#customizing-head-metadata

**Reply by jcstein** [reply]
got it. thanks again @slorber
